File: java/users-micronaut-java/src/main/java/com/talentwunder/controller/UserController.java
package com.talentwunder.controller;

import com.talentwunder.dto.UserDTO;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

import java.util.List;

@Controller("/users")
public class UserController {

    @Get("/")
    List<UserDTO> get() {
        return List.of(new UserDTO("f08ec591-ac64-4425-b8b2-f92b1a1e28fb", "test@talentwunder.com"));
    }
}

File: kotlin/users-micronaut-kotlin/src/main/kotlin/com/talentwunder/controller/UserController.kt
package com.talentwunder.controller

import com.talentwunder.dto.UserDTO
import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get

@Controller("/users")
class UserController {

    @Get("/")
    fun get(): List<UserDTO> {
        return listOf(UserDTO("ff560988-a020-4afa-b455-37d19d649aae", "test@talentwunder.com"))
    }
}
